### ISSUE TITLE
CC007 false positive: Not MatchesPath condition flagged

### ISSUE THREAD

**Issue by guycall** [apigee/apigeelint#585]
I have the condition:

```
<Step>
  <Condition>proxy.pathsuffix Not MatchesPath "/somepath"</Condition>
  <Name>SomePolicy</name>
</Step>
```

Apigeelint v2.75.0 flags as an error:

`Invalid expression is invalid. Poisition 18, SyntaxError: Expected .... but "N" found.`

I have checked with the Apigee Debug feature and the condition works as expected when deployed.

**Comment by DinoChiesa** [comment]
Yuck, really?   `Not MatchesPath` ? 

I didn't realize that form is valid. 

To my eye, `Not (proxy.pathsuffix MatchesPath "/somepath")` seems more clear. But if Apigee supports it. . . .  apigeelint should pass it.

**Comment by guycall** [comment]
Thank you. Resolved.
